open-api/rest-catalog-open-api.yaml

```diff
@@ -212,6 +212,34 @@ paths:
           schema:
             type: string
           example: "accounting%1Ftax"
+        - name: pageToken
+          in: query
+          description:
+            Allows clients to make use of pagination by signaling to the service that they would
+            prefer requests to be paginated to a “reasonable” number of results.
+            
+            New Clients always start the request by sending a required empty “pageToken” e.g. GET /tables?continuationToken=””
+            Servers supporting pagination would recognize pagination is requested due to the presence of the pageToken
+            and honor the contracts specified above by returning a non-empty continuation token
```

Comment:
can we standardize the terminology to page token, instead of using page token and continuation token interchangeably?

Reply:
Ack will fix this.